tests/unit_tests/test_wave_06.py

```diff
@@ -109,8 +111,20 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) ==3
+    
+    assert item_a in tai.inventory          # Decor(2.0)
+    assert item_b in tai.inventory          # Electronics(4.0) 
+    assert item_f in tai.inventory          # Jesse  Clothing(4.0)
+
+    assert item_d in jesse.inventory        # Clothing(2.0) 
+    assert item_e in jesse.inventory        # Decor(4.0) 
+    assert item_c in jesse.inventory        # Tai Decor(4.0)
```

Comment:
Great assertions for all the items in the inventories to ensure all of the items we expect are present in the correct locations!